client/finality-grandpa/src/lib.rs

```diff
@@ -68,7 +68,9 @@ use sc_client_api::{
 	StorageProvider, TransactionFor,
 };
 use sc_consensus::BlockImport;
-use sc_network_common::protocol::ProtocolName;
+use sc_network_common::{
+    protocol::ProtocolName,
+};
```

Comment:
`cargo +nightly fmt` should fix the formatting and make the CI happy

Reply:
Thanks, fixed